tests/integration/account/test_account.py

```diff
@@ -39,16 +41,38 @@ def test_event_list():
         .rstrip()
     )
     lines = res.splitlines()
-
-    event_id = lines[1].split(",")[0]
-
     headers = ["entity.label", "username"]
     assert_headers_in_lines(headers, lines)
-    return event_id
 
 
-def test_event_view():
-    event_id = test_event_list()
+@pytest.fixture
+def test_event_id():
```

Comment:
Can you name the fixture something not followed by `test`? Just to keep it consistent with rest of the fixtures in the suite